tests/integration/chain_setup.go

```diff
@@ -0,0 +1,120 @@
+package integration
+
+import (
+	"context"
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	buildertypes "github.com/skip-mev/pob/x/builder/types"
+	interchaintest "github.com/strangelove-ventures/interchaintest/v7"
+	"github.com/strangelove-ventures/interchaintest/v7/chain/cosmos"
+	"github.com/strangelove-ventures/interchaintest/v7/ibc"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zaptest"
+)
+
+// ChainBuilderFromChainSpec creates an interchaintest chain builder factory given a ChainSpec
+// and returns the associated chain
+func ChainBuilderFromChainSpec(t *testing.T, spec *interchaintest.ChainSpec) ibc.Chain {
```

Comment:
nit: I would make all of these inherit the the test suite. Why are you passing `testing.T` directly?